### PR TITLE
Add a contributing documentation

### LINKED ISSUE #14: Contributing documentation
## Expected behaviour
I want to find a page of documentation on how to contribute to this project.

## Current behavior
I've no one information for the moment.

## Potential solution
1. Add a page `How to contribute` in the documentation `ReadTheDocs`

 - How to find an issue (waffle, github)
 - How to commit 
 - Test Driven Development
 - Inform you work on an issue by commenting this one before you begin to work on it (or assign yourself if you're a member of the organization)
 - How to name your branch (prefix + snake_case, ex: `enhancement-create_new_cell`, `fix-pep8_standard_on_cell_model`)

2. Modify the template of `Pull Request` to make a link on the contribution page in place of the `Contributing.md` who doesn't exist

### PR DESCRIPTION
**Pull request by DarkFranX**
| Q                                                      | R
| ------------------------------------------ | -------------------------------------------
| Type of contribution ?                      | [Documentation]
| Tickets (_issues_) concerned               | fixed #14

---

##### What have you changed ?
Add a contribution document describing guidelines for new contributors

##### How did you change it ?
New file named docs/CONTRIBUTING.md

Verification :
 -  [X] My branch name respect the standard defined in `CONTRIBUTING.md`
 -  [X] This Pull-Request fully meets the requirements defined in the issue
 -  [X] I added or modified the attached tests

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+This section guides you through submitting a bug report for Volontaria. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out [the required template](.github/ISSUE_TEMPLATE.md), the information it asks for helps us resolve issues faster.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). After you've determined [which repository](#atom-and-packages) your bug is related to, create an issue on that repository and provide the following information by filling in [the template](.github/ISSUE_TEMPLATE.md).
```
Comment (DarkFranX) [inline docs/CONTRIBUTING.md]: atom

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+* If the problem started happening recently, **can you reproduce the problem in an older version?** What's the most recent version in which the problem doesn't happen?
+* **Can you reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens.
+
+Include details about your configuration and environment:
+
+* **What's the name and version of the OS you're using**?
+* **Are you running in a virtual machine?** If so, which VM software are you using and which operating systems and versions are used for the host and the guest?
+* **Which keyboard layout are you using?** Are you using a US layout or some other layout?
+
+### Suggesting Enhancements
```
Comment (DarkFranX) [inline docs/CONTRIBUTING.md]: Use feathub:
- Verify if suggestion is already created 
- If a suggestion similar to yours, just comment on the existing suggestion
- Else create a new suggestion (link in readme)

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). After you've determined [which repository](#atom-and-packages) your enhancement suggestion is related to, create an issue on that repository and provide the following information:
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots and animated GIFs** which help you demonstrate the steps or point out the part of Atom which the suggestion is related to. You can use [this tool](http://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux.
```
Comment (DarkFranX) [inline docs/CONTRIBUTING.md]: Not sure, check with feathub

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+* **Explain why this enhancement would be useful** to most Volontaria users.
+* **Specify the name and version of the OS you're using.**
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to Volontaria? You can start by looking through these `beginner` and `help-wanted` issues:
+
+* [Beginner issues][beginner] - issues which should only require a few lines of code, and a test or two.
```
Comment (DarkFranX) [inline docs/CONTRIBUTING.md]: good first issue

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+* [Help wanted issues][help-wanted] - issues which should be a bit more involved than `beginner` issues.
+
+### Pull Requests
+* Branches naming convention: prefix + snake_case, ex: enhancement-create_new_cell, fix-pep8_standard_on_cell_model
+* Fill in [the required template](.github/PULL_REQUEST_TEMPLATE.md)
```
Comment (DarkFranX) [inline docs/CONTRIBUTING.md]: automatically on github

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+    $ git commit -m "A brief summary of the commit"
+    >
+    > "A paragraph describing what changed and its impact."
+
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+* Limit the first line to 72 characters or less
+* Reference issues and pull requests liberally after the first line
+* When only changing documentation, include `[ci skip]` in the commit description
```
Comment (DarkFranX) [inline docs/CONTRIBUTING.md]: remove

File: docs/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+      "key1": 1,
+      "key2": 2,
+      "key3": 3
+    }
+    ```
+  * This is open source software. Consider the people who will read your code, and make it look nice for them. It's sort of like driving a car: Perhaps you love doing donuts when you're alone, but with passengers the goal is to make the ride as smooth as possible.
+
+## Additional Notes
+
+### Issue and Pull Request Labels
```
Comment (DarkFranX) [inline docs/CONTRIBUTING.md]: pull request (no label)

File: docs/CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+* **Verify if the same suggestion has already been created on Feathub** so that you do not create a duplicate.
+* **If a suggestion similar to yours**, comment on the existing suggestion instead of creating a new one.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked in [Volontaria Feathub page](https://feathub.com/Volontaria/API-Volontaria).
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
```
Comment (RignonNoel) [inline docs/CONTRIBUTING.md]: Je viens de test et ca fonctionne bien

**Pull request merged**